.circleci/config.yml

```diff
@@ -158,7 +158,7 @@ commands:
           name: "restore e2e cache"
           keys:
             - v1-e2e-cache-{{ .Branch }}-{{ checksum "~/workbench/e2e/yarn.lock" }}
-            - v1-e2e-cache-master
+            - v1-e2e-cache-{{ .Branch }}
```

Comment:
Why this change? I think I prefer master here. As I understand it, using the branch name here only helps in the following scenario:

- I'm working on a feature branch
- I push one set of changes to yarn.lock
- On that same feature branch, I push a different set of changes to yarn.lock

In this scenario, unless you are making a ton of changes to the lockfile, you're probably getting the same performance as just restoring from the master cache.

Alternatively, if I'm working on a new feature branch and change yarn.lock (more likely), my cache hit may be worse in this model, since I'd pull in any recent cache entry, possibly from an unrelated feature branch.

Reply:
yeah you're right. I didn't think about that. Reverted change. Also, I realized same mistake I made in e2e cache.

Reply:
I spoke too soon about e2e cache mistake. I had `v1-e2e-cache-master`, nothing to fix.